tests/data/test_btanalysis.py

```diff
@@ -105,6 +105,7 @@ def test_load_trades(default_conf, mocker):
     load_trades("DB",
                 db_url=default_conf.get('db_url'),
                 exportfilename=default_conf.get('exportfilename'),
+                no_trades=False
```

Comment:
I mainly noticed the above comments via default because of this ... so i'd probably revert this line (and line 120)

Reply:
Well i guess it's not needed but it's also not harming.